config.py

```diff
@@ -23,16 +23,16 @@
     "connection_name": os.environ.get("CONNECTION_NAME"),
 }
 # For socket based connection
-SQLALCHEMY_DATABASE_URI = (
-    "postgresql://%(user)s:%(password)s@/%(database)s?host=%(connection_name)s/"
-    % POSTGRES
-)
-if FLASK_ENV == "development":
-    # For TCP based conneciton
+if FLASK_ENV == "staging":
     SQLALCHEMY_DATABASE_URI = (
-        "postgresql://%(user)s:%(password)s@%(host)s:%(port)s/%(database)s" % POSTGRES
+        "postgresql://%(user)s:%(password)s@/%(database)s?host=%(connection_name)s/"
+        % POSTGRES
     )
 
+SQLALCHEMY_DATABASE_URI = (
```

Comment:
Won't this overwrite the values set on line no. 27?

Reply:
Yes, fixed now